js/ui/drop_down_button.js

```diff
@@ -451,9 +449,17 @@ const DropDownButton = Widget.inherit({
 
     _popupShowingHandler() {
         this.option('opened', true);
-        this.setAria({
-            expanded: true,
-            owns: this._popupContentId
+        this._setAriaExpanded(true);
+    },
+
+    _setAriaExpanded(value) {
+        const ariaElements = [ ...this._buttonElements, this.$element() ];
```

Comment:
let's cache not only button elements, but all "aria expanded elements" to avoid excess destucturizations?